src/main/kotlin/net/atos/zac/app/zaak/converter/RestZaakOverzichtConverter.kt

```diff
@@ -27,6 +27,8 @@ class RestZaakOverzichtConverter @Inject constructor(
     private val policyService: PolicyService,
     private val zrcClientService: ZrcClientService,
 ) {
+
+    @Deprecated("user parameter is to be removed when we switch to paging")
```

Comment:
This is a comment on future changes, not a method deprecation, right? then I would probably remove it completely.